robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowBundle.java

```diff
@@ -1,50 +1,11 @@
 package org.robolectric.shadows;
 
 import android.os.Bundle;
-import org.robolectric.annotation.Implementation;
 import org.robolectric.annotation.Implements;
-import org.robolectric.annotation.RealObject;
-import org.robolectric.util.ReflectionHelpers;
-
-import java.util.Map;
 
 /**
  * Shadow for {@link android.os.Bundle}.
  */
 @Implements(Bundle.class)
 public class ShadowBundle {
```

Comment:
No good reason. I'll remove it as part of fixing the remainder of the Travis failures.